src/ui/edittagdialog.cpp

```diff
@@ -833,27 +834,33 @@ void EditTagDialog::FetchTagSongChosen(const Song& original_song,
   const QString filename = original_song.url().toLocalFile();
 
   // Find the song with this filename
-  auto data_it =
-      std::find_if(data_.begin(), data_.end(), [&filename](const Data& d) {
-        return d.original_.url().toLocalFile() == filename;
-      });
-  if (data_it == data_.end()) {
-    qLog(Warning) << "Could not find song to filename: " << filename;
-    return;
+  int id;
```

Comment:
For finding out if the song is currently displayed (see below).
(Otherwise a range-based for loop could be used here)